tools/crd-bumper/pkg/controllers.py

```diff
@@ -283,6 +284,23 @@ def _update_spoke_conversion(self, kinds, path, this_api):
         fu.replace_in_file(f"_{self._prev_ver}_", f"_{self._new_ver}_")
         fu.store()
 
+    def _find_spoke_carryovers(self, path, this_api, final_msgs):
+        """Search the conversion.go in each pre-existing spoke to find any code
+        that is marked for carry-over to new spokes.
+        """
+
+        if this_api == self._prev_ver:
+            # Search only the established spokes.
+            return
+
+        fu = FileUtil(self._dryrun, path)
+        lines = fu.find_all_in_file("+crdbumper:carryforward:begin")
```

Comment:
does it do anything with `+crdbumper:carryforward:end` or is that just for the user to know when to stop? They will have to manually copy, right?

Reply:
Right now the "carryforward:end" in the conversion code helps the human reader. In the future the tool ought to be able to use it to grab that chunk of code and move it.